.mergify.yml

```diff
@@ -38,6 +38,24 @@ queue_rules:
               - "status-success=ci/centos/upgrade-tests-cephfs"
               - "status-success=ci/centos/upgrade-tests-rbd"
               - "status-success=DCO"
+          - and:
+              - base=release-v3.5
+              - "status-success=codespell"
+              - "status-success=multi-arch-build"
+              - "status-success=go-test"
+              - "status-success=commitlint"
+              - "status-success=golangci-lint"
+              - "status-success=mod-check"
+              - "status-success=lint-extras"
+              - "status-success=ci/centos/mini-e2e-helm/k8s-1.21"
+              - "status-success=ci/centos/mini-e2e-helm/k8s-1.22"
+              - "status-success=ci/centos/mini-e2e-helm/k8s-1.23"
+              - "status-success=ci/centos/mini-e2e/k8s-1.21"
+              - "status-success=ci/centos/mini-e2e/k8s-1.22"
+              - "status-success=ci/centos/mini-e2e/k8s-1.23"
+              - "status-success=ci/centos/upgrade-tests-cephfs"
+              - "status-success=ci/centos/upgrade-tests-rbd"
+              - "status-success=DCO"
```

Comment:
why this is needed isn't it already converted in the above rule?

Reply:
> why this is needed isn't it already converted in the above rule?

if we change rules for devel branch in the future, we will need to create the above set of rules again ?

Reply:
yes, change it when required?

Reply:
it is better to keep seperate rules for maintained branches, isnt it ? and keep devel intact @Madhu-1 ?

Reply:
I am fine with either way, if we dont need release specific rules , I can remove existing `release-3.4` set too. wdyt @Madhu-1 @Rakshith-R

Reply:
release-3.4 is too old and new tests are added in devel branch so you cannot remove it. am fine with this one.